workers/unity/Packages/com.improbable.gdk.core/Worker/WorldCommandsReceivedStorage.cs

```diff
@@ -71,5 +83,114 @@ public void AddResponse(WorldCommands.EntityQuery.ReceivedResponse response)
         {
             return new ReceivedMessagesSpan<WorldCommands.EntityQuery.ReceivedResponse>(entityQueryResponses);
         }
+
+        ReceivedMessagesSpan<WorldCommands.CreateEntity.ReceivedResponse>
+            IDiffCommandResponseStorage<WorldCommands.CreateEntity.ReceivedResponse>.GetResponse(long requestId)
+        {
+            if (!createEntitySorted)
+            {
+                createEntityResponses.Sort(comparer);
+                createEntitySorted = true;
+            }
+
+            var responseIndex = createEntityResponses.GetResponseIndex(requestId);
+            if (responseIndex < 0)
```

Comment:
Is this valid behaviour to run into? Should it display any log?

Reply:
Yes this just means you are polling for a response that hasn't arrived yet. There is a question as to whether or not we want this API or a different one.